### PR TITLE
GATK 4.1.4.0 Dev

### PR DESCRIPTION
**Pull request by bshifaw**
Update WDL to  GATK 4.1.4.0 release.  
- WDL now written in WDL 1.0 -- enabling call-caching on Terra 
- Oncotator is no longer an option -- the workflow only supports Functotator now

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,10 +36,10 @@ Used to validate mutect2 workflow.
 - False Positive VCF files and its index with summary  
      
 ### Software version requirements :
-- GATK4.1.2.0 
+- GATK4.1.3.0
```
Comment (bshifaw) [inline README.md]: change to GATK4.1.4.0

File: mutect2.inputs.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "Mutect2.gatk_docker": "broadinstitute/gatk:4.1.3.0",
```
Comment (bshifaw) [inline mutect2.inputs.json]: change to gatk:4.1.4.0

File: mutect2_pon.inputs.json [hunk 3]
```diff
@@ -1,20 +1,16 @@
...
 	"Mutect2_Panel.ref_dict":"gs://gatk-best-practices/somatic-b37/Homo_sapiens_assembly19.dict",
 	"Mutect2_Panel.scatter_count":10,
 
-	"##_COMMENT4": "Secondary resources",
 	"Mutect2_Panel.intervals":"gs://gatk-best-practices/somatic-b37/whole_exome_agilent_1.1_refseq_plus_3_boosters.Homo_sapiens_assembly19.baits.interval_list",
 	"Mutect2_Panel.Mutect2.variants_for_contamination":"gs://gatk-best-practices/somatic-b37/small_exac_common_3.vcf",
 	"Mutect2_Panel.gnomad":"gs://gatk-best-practices/somatic-b37/af-only-gnomad.raw.sites.vcf",
 
-	"##_COMMENT6": "Docker",	
 	"Mutect2_Panel.gatk_docker":"broadinstitute/gatk:4.1.2.0"
```
Comment (bshifaw) [inline mutect2_pon.inputs.json]: change to gatk:4.1.4.0

File: mutect2.inputs.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (davidbenjamin) [inline mutect2.inputs.json:1]: @bshifaw I reviewed these inputs as you requested and they are fine.

File: mutect2_pon.wdl [hunk 5]
```diff
@@ -9,33 +11,53 @@
 #  m2_extra_args: additional command line parameters for Mutect2.  This should not involve --max-mnp-distance,
 #  which the wdl hard-codes to 0 because GenpmicsDBImport can't handle MNPs
 
-import "https://raw.githubusercontent.com/gatk-workflows/gatk4-somatic-snvs-indels/2.5.0/mutect2_nio.wdl" as m2
+#import "mutect2.wdl" as m2
+
+import "https://raw.githubusercontent.com/gatk-workflows/gatk4-somatic-snvs-indels/dev/mutect2.wdl" as m2
```
Comment (bshifaw) [inline mutect2_pon.wdl]: Change "dev" to next release number

**Pull request merged**